src/vmm_mad/remotes/kvm/detach_disk

```diff
@@ -29,9 +29,10 @@ DETACH_PARAMS="--domain $DOMAIN --target $TARGET"
 exec_and_log "virsh --connect $LIBVIRT_URI detach-disk $DETACH_PARAMS" \
     "Could not detach $TARGET from $DOMAIN"
 
-virsh --connect $LIBVIRT_URI dumpxml $DOMAIN | grep $TARGET > /dev/null 2>&1
+virsh --connect $LIBVIRT_URI dumpxml $DOMAIN | \
+    xmllint -xpath "//disk/target[@dev='$TARGET']" >/dev/null 2>&1
```

Comment:
This will adds `xmllint` to dependencies, wouldn't be better to use `/var/tmp/one/datastore/xpath.rb` instead?

Reply:
The following reasons:
* `xmllint` is part of the `libxml2` on which several `libvirt` sub-packages depends
* the exit code of `xpath.rb` could not be used. I hate adding unnecessary piping/processing just to get an exit status.

Reply:
... and `augeas` ;-)
So it is already available due to required dependencies.

Reply:
True, so I also prefer using `xmllint`. One small thing, there seems to be missing `-` (hyphen sign) as a last xmllint parameter to even process the stdin.

Reply:
hm. stupid typo :)
how do you prefer - to push second commit or edit and force-push?

Reply:
Doesn't really matter, you can force-push or add another commit, which will be squashed when merging.